src/actionLoading.ts

```diff
@@ -82,17 +81,30 @@ export const loadAllActions = (actionDir: string, envConfig: any): Action[] => {
 
 /**
  * Extracts the host from the action definition if present or from the
- * environment configuration. If the http protocol is not specified explicitly,
- * https is assumed.
+ * environment configuration. In case of REST actions, if the http protocol is
+ * not specified explicitly, https is assumed. In case of WebSocket actions, if
+ * the ws protocol is not specified explicitly, wss is assumed.
  * @param actionDef The action definition
  * @param envConfig The environment configuration
  */
-const getHost = (actionDef: any, envConfig: any): string => {
-    if (actionDef.service.startsWith('http')) {
-        return actionDef.service;
+const getHost = (actionDef: any, envConfig: any): string | void => {
```

Comment:
```suggestion
const getHost = (actionDef: any, envConfig: any): string | undefined => {
```

Reply:
Doing this would also require us to validate user input (i.e. the used type in the action definition) before constructing the action. This is the correct thing to do, but out of scope here. I'll create an issue for that.

Reply:
done